docs/unified-test-documentation/dasharo-compatibility/340-docking-station-net-interface.md

```diff
@@ -88,8 +86,6 @@ Ethernet port can be obtained on Windows 11.
 1. Power on the DUT.
 1. Boot into the system.
 1. Log into the system by using the proper login and password.
```

Comment:
As above. Please review all documentation and determine if such information would need to be added.

Reply:
I have reviewed and added information where needed